go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/riandyrn/otelchi
+module github.com/tjefferson08/otelchi
```

Comment:
Changing the module name will break imports. Please keep it as `github.com/riandyrn/otelchi`.

Reply:
Of course! I'm new to the go ecosystem; this was a failed attempt to make my forked code available as a dependency for testing purposes